### PR TITLE
Feature/cypress

### PR DESCRIPTION
**Pull request by joefitter**
* Updated to use Cypress for integration tests
* Added integration tests for org mapper

### CODE REVIEW

File: .github/workflows/deploy-dev.yml [hunk 1]
```diff
@@ -156,7 +156,7 @@ jobs:
           cache: 'npm'
           cache-dependency-path: ui/package-lock.json
       - run: npm ci
-      - run: npm run test:integration -- --baseUrl https://dev.nhs.marvell-consulting.com
+      - run: CYPRESS_baseUrl=https://dev.nhs.marvell-consulting.com npm run test:integration
```
Comment (lennym) [inline .github/workflows/deploy-dev.yml:159]: Sweet Jesus, such case.

File: ui/cypress/plugins/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+// the project's config changing)
+
+/**
+ * @type {Cypress.PluginConfig}
+ */
+// eslint-disable-next-line no-unused-vars
+module.exports = (on, config) => {
+  // `on` is used to hook into various events Cypress emits
+  // `config` is the resolved Cypress config
+  config.baseUrl = process.env.BASE_URL || config.baseUrl;
```
Comment (lennym) [inline ui/cypress/plugins/index.js:22]: https://media1.giphy.com/media/nUECVDZSR1xmpxBcHS/giphy.gif

File: ui/cypress/fixtures/example.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
[the remark this answers is not in the document]
Comment (joefitter) [reply ui/cypress/fixtures/example.json:5]: I have a couple of times, but I think it keeps coming back when you run the interactive tool

[the remark this answers is not in the document]
Comment (lennym) [reply ui/cypress/fixtures/example.json:5]: Add it to `.gitignore`?

[the remark this answers is not in the document]
Comment (joefitter) [reply ui/cypress/fixtures/example.json:5]: herr derp

**Review by lennym — APPROVED** [review]

**Pull request merged**